dhanwade12-bristol.md

```diff
@@ -0,0 +1,12 @@
+# Bristol, UK
+
+###Things to do
+Ballon Fiesta in Aug
+
+###Places to eat
+Nicholas market
+Gloucester Road
+
+
```

Comment:
Excellent! So you are happy for me to merge Bristol with the main branch :smiley: